README.md

```diff
@@ -122,6 +122,36 @@ your container. This will allow you to login to the root account remotely. Local
 connections will still not require a password.
 
 
+Usage on Atomic host
+---------------------------------
+Systems derived from projectatomic.io usually include `atomic` command that is
+used to run containers except other tasks.
+
+To install a new service `mysql1` based on this image on such a system, run:
+
+```
+$ atomic install -n mysql1 --opt2='-e MYSQL_USER=user` -e MYSQL_PASSWORD=secretpass -e MYSQL_DATABASE=db1 -p 3306:3306' THIS_IMAGE
+```
+
+Then to run the service, use the standard `systemctl` call:
+
+```
+$ systemctl start mysqld1.service
```

Comment:
mysqld1 seems better, it is similar to main daemon name from mysql. So I'll fix the other occurrences.